gbfs.md

```diff
@@ -1423,11 +1448,175 @@ Field Name | REQUIRED | Type | Defines
           }
         }
       ]
-    }
+    },
+    "global_rules": [
+      {
+        "ride_start_allowed": false,
+        "ride_end_allowed": false,
+        "ride_through_allowed": true
+      }
+    ]
   }
 }
 ```
 
+##### Polygon Overlap Examples
+
+The following polygon diagram will be used in a number of examples below of geofencing rule resolution.
+
+![Geofencing Example Diagram with two overlapping polygons A and B, and areas a, b, and ab.](geofencing_example.svg)
```

Comment:
I tried to look at this using a dark mode browser and the text in the diagrams was illegible because of the transparent background. Would be better if they were on white or the text color was changed. Also, we should probably set up an /images directory to keep things neat and clean as more images are added to the repo in the future.

Reply:
To keep things simple, I switched the image to a white background.  I also moved it to a new images/ sub-directory.